pulsar-broker/src/test/java/org/apache/pulsar/client/impl/NegativeAcksTest.java

```diff
@@ -321,7 +321,7 @@ public void testNegativeAcksDeleteFromUnackedTracker() throws Exception {
         negativeAcksTracker.close();
     }
 
-    @Test(timeOut = 10000)
```

Comment:
+1 to the previous question.

Reply:
Locally run takes about 8s sometimes, so remove this.

Reply:
Okay. Anyway, this change does not makes it a blocker. This fix LGTM.